nodeup/pkg/model/kube_proxy.go

```diff
@@ -124,9 +124,39 @@ func (b *KubeProxyBuilder) buildPod() (*v1.Pod, error) {
 		}
 	}
 
-	cpuRequest, err := resource.ParseQuantity(c.CPURequest)
-	if err != nil {
-		return nil, fmt.Errorf("Error parsing CPURequest=%q", c.CPURequest)
+	resourceRequests := v1.ResourceList{}
+	resourceLimits := v1.ResourceList{}
+
+	if c.CPURequest != "" {
```

Comment:
@horaceheaven  We need to revert the CPURequest logic to maintain the existing behavior.